### ISSUE TITLE
Dead link in menu of arrow-kt.io

### ISSUE THREAD

**Issue by rybak** [arrow-kt/arrow#2523]
#### Steps to reproduce
1. Go to https://arrow-kt.io/docs/typeclasses/intro/
2. On the left, click on "Docs"

#### Actual result
* Link leads to https://arrow-kt.io/docs/apidocs/arrow-core/ which doesn't exist (404)

#### Expected result
* Link leads to an existing location

-----

Side note: I couldn't find separate repository for the website at https://github.com/arrow-kt, I assume that https://github.com/arrow-kt/arrow/issues is the correct place for reporting issues with the website. Please correct me if I'm wrong.

**Comment by fo2rist** [comment]
Check if that's a sufficient fix https://github.com/arrow-kt/arrow/pull/2664
I'm not the author, but it seems to be similar to what it should to be

**Comment by nomisRev** [comment]
Thank you for the PR @fo2rist 🙏

**Comment by rybak** [comment]
I believe that the original bug got resolved by PR #2477, since its diff includes [this change in file `arrow-site/docs/_data/sidebar-core.yml`](https://github.com/arrow-kt/arrow/pull/2477/files#diff-2a5a1dae2765d493b25e4e80bc123175af8b23ddea11999b12b3f6e74178750e):

```diff
-  - title: Docs
-    url: /apidocs/arrow-core/
+  - title: Api Docs
+    url: /apidocs/arrow-core/arrow.core
```

**Comment by fo2rist** [comment]
Thanks! You're right, I've mixed up two different things.
Will remove the link between PR and the issue.
